Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/WriterTests/CloudFormationWriterTests.cs

```diff
@@ -498,6 +499,228 @@ public void CodeUriTest(CloudFormationTemplateFormat templateFormat)
             Assert.Equal("serverlessApp", templateWriter.GetToken<string>($"{propertiesPath}.CodeUri"));
         }
 
+        #region CloudFormation template description
+
+        /// <summary>
+        /// Tests that the CloudFormation template's "Description" field is set
+        /// correctly for a entirely new template.
```

Comment:
nit: use "an" instead of "a" before "entirely"

Reply:
Good eye, fixed via amending the commit.